src/main/java/org/jenkinsci/plugins/oic/OicUserProperty.java

```diff
@@ -19,22 +20,22 @@ public static class Descriptor extends UserPropertyDescriptor {
 		@Override
 		public UserProperty newInstance(User user) {
 			LOGGER.fine("OicUserPropertyDescriptor.newInstance called, user:" + user);
-			return new OicUserProperty(user.getId(), new GrantedAuthority[0]);
+			return new OicUserProperty(user.getId(), new ArrayList<GrantedAuthority>());
 		}
 
 		@Override
 		public String getDisplayName() {
-			return Messages.openid_connect_user_property();
+			return "OpenID Connect user property";
```

Comment:
FYI, this is just removing translations, seems like an odd choice unless you are adding new strings.